packages/world/src/modules/core/implementations/AccessManagementSystem.sol

```diff
@@ -71,4 +71,23 @@ contract AccessManagementSystem is System, LimitedCallContext {
     // Grant access to new owner
     ResourceAccess._set(namespaceId, newOwner, true);
   }
+
+  /**
+   * @notice Renounces ownership of the given namespace
+   * @dev Requires the caller to own the namespace. Revoke ResourceAccess for previous owner
+   * @param namespaceId The ID of the namespace to transfer ownership.
+   */
+  function renounceOwnership(ResourceId namespaceId) public virtual onlyDelegatecall {
+    // Require the namespace ID to be a valid namespace
+    requireNamespace(namespaceId);
+
+    // Require the caller to own the namespace
+    AccessControl.requireOwner(namespaceId, _msgSender());
+
+    // Delete namespace owner
+    NamespaceOwner._deleteRecord(namespaceId);
+
+    // Revoke access from old owner
+    ResourceAccess._deleteRecord(namespaceId, _msgSender());
+  }
```

Comment:
should we just call `transferOwnership` with `address(0)` here? Similar to the Ownable lib: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/e5c63635e3508a8d9d0afed091578cc4bb59a9c7/contracts/access/Ownable.sol#L77

Reply:
Could transfer to address 0 and then delete the resource access record. No strong opinion though.

Reply:
that would basically be the same logic but a few extra writes, wouldn't it?

Reply:
I think this is fine how it is, but I did make a code suggestion below for completeness with other audit fixes (adding namespace existence check for better reverts)